### DISCUSSION TITLE
Table Design

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#9410] [Help] [answered]
Hi,

I have a question about components you can add to a table.
When you add, for example, a button to the table, it sticks to the left.
Is there a way to put it on the right side?

Sincerely,
K

**Comment by melohagan** [comment, marked as answer]
Not currently @kasgithub8 

There is an open issue here: https://github.com/Budibase/budibase/issues/5005

It may also be worth having a look in the Discord server: https://discord.gg/NX6sQCYv

There's a **components** channel where the community has been developing various custom components; there may be something there, or you could get help building something yourself.

More on plugins here: https://docs.budibase.com/docs/custom-plugin

**Reply by Simoon-F** [reply]
Do you have any plans to modify this, except for the development components？THX～
